docs/admin/logging/overview.md

```diff
@@ -130,6 +131,10 @@ By default, Docker uses the first 12 characters of the container ID to tag log m
 Refer to the [log tag option documentation](log_tags.md) for customizing
 the log tag format.
 
+`syslog-format` configures syslog protocol version to use when logging,
+Allowed value is rfc5424 in order to use RFC5424 syslog protocol. 
+Default is RFC3164 incompatible syslog protocol.
```

Comment:
actually lets omit "default" string to make it compatible with the "syslog-address" parameter which defaults to local unix domain socket, if absent, but there is no other way to specify default, so again : 

1) without specifying anything lets keep previous behaviour (the UnixFormatter)
2) rfc5424 - RFC5424
3) rfc3164 - RFC3164

Agreed No2 :)